docs/platforms/android/configuration/options.mdx

```diff
@@ -303,35 +303,15 @@ Set this boolean to `false` to disable tracing for `OPTIONS` requests. This opti
 
 ## Profiling Options
 
-<ConfigKey name="profiles-sample-rate">
-
-A number between `0` and `1`, controlling the percentage chance a given profile will be sent to Sentry. (`0` represents 0% while `1` represents 100%.) Applies only to sampled transactions created in the app. Either this or <PlatformIdentifier name="profiles-sampler" /> must be defined to enable transaction profiling. Don't set it if you want to use <PlatformLink to="/profiling/#continuous-profiling">Continuous Profiling</PlatformLink>.
-
-</ConfigKey>
-
-<ConfigKey name="profiles-sampler">
-
-A function responsible for determining the percentage chance a given profile will be sent to Sentry. It will automatically be passed information about the transaction and the context in which it's being created, and must return a number between `0` (0% chance of being sent) and `1` (100% chance of being sent). Can also be used for filtering profiles, by returning 0 for those that are unwanted. Either this or <PlatformIdentifier name="profiles-sample-rate" /> must be defined to enable transaction profiling. Don't set it if you want to use <PlatformLink to="/profiling/#continuous-profiling">Continuous Profiling</PlatformLink>.
-
-</ConfigKey>
-
-<ConfigKey name="enable-app-start-profiling">
-
-A boolean value that determines whether the app start process will be profiled. When true, the startup process, including ContentProviders, Application and first Activity creation, will be profiled. Note that <PlatformIdentifier name="profiles-sample-rate" /> or <PlatformIdentifier name="profiles-sampler" /> must be defined.
-
-</ConfigKey>
-
-## Continuous Profiling Options
-
-<Alert>
+<Alert title="Note">
 
-This feature is experimental and may have bugs.
+Profiling requires SDK versions 8.7.0 or higher. Lower versions can use the <PlatformLink to="/configuration/options/#legacy-profiling-options">legacy profiling</PlatformLink>.
```

Comment:
We refer to "legacy profiling" as "transaction-based Profiling" in other docs, so we should use that terminology here too and in other places in these docs.